content/en/serverless/step_functions/installation.md

```diff
@@ -0,0 +1,191 @@
+{{% /tab %}}
+{{% tab "Custom" %}}
+
+1. Enable all logging for your Step Function. In your AWS console, open your state machine. Click *Edit* and find the Logging section. There, set *Log level* to `ALL` and enable the *Include execution data* checkbox.
+
+2. Ensure you have deployed the [Datadog Lambda Forwarder][1], a Lambda function that ships logs from AWS to Datadog, and that you are using v3.66.0+. You may need to [update your Forwarder][2].
+
+   Take note of your Forwarder's ARN.
+
+3. Subscribe CloudWatch logs to the Datadog Lambda Forwarder. To do this, you have two options:
+   - **Datadog-AWS integration** (recommended)
+     1. Ensure that you have set up the [Datadog-AWS integration][4].
+     2. In Datadog, open the [AWS integration tile][5], and view the *Configuration* tab.
+     3. On the left, select the AWS account where your Step Function is running. Open the *Log Collection* tab.
+     4. In the *Log Autosubscription* section, under *Autosubscribe Forwarder Lambda Functions*, enter the ARN of your Datadog Lambda Forwarder, as noted previously. Click *Add*.
+     5. Toggle on *Step Functions CloudWatch Logs*. Changes take 15 minutes to take effect.
+
+     **Note**: Log Autosubscription requires your Lambda Forwarder and Step Function to be in the same region.
+
+   - **Manual**
+     1. Ensure that your log group name has the prefix `/aws/vendedlogs/states`. 
+     2. Open your AWS console and go to your Datadog Lambda Forwarder. In the *Function overview* section, click on *Add trigger*.
+     3. Under *Add trigger*, in the *Trigger configuration* section, use the *Select a source* dropdown to select `CloudWatch Logs`.
+     4. Under *Log group*, select the log group for your state machine. For example, `/aws/vendedlogs/states/my-state-machine`.
+     5. Enter a filter name. You can choose to name it "empty filter" and leave the *Filter pattern* box blank.
+4. Enable tracing on your Step Functions. Set the environment variable `DD_TRACE_ENABLED` to `true`.
+<!-- Where is this environment variable set? -->
+5. Set up tags. Open your AWS console and go to your Step Functions state machine. Open the *Tags* section and add `env:<ENV_NAME>` and `service:<SERVICE_NAME>` tags. The `env` tag is required to see traces in Datadog. The `service` tag defaults to the state machine's name.
+6. For Node.JS and Python runtimes, you can link your Step Function traces to Lambda traces. On the Lambda Task, set the `OutputPath` and `Parameters` keys with the following: 
+
+   ```json
+   "OutputPath": "$.Payload",
+   "Parameters": {
+     "Payload.$": "States.JsonMerge($$, $, false)",
+     "FunctionName": "${lambdaArn}"
+   }
+   ```
+
+   The `JsonMerge` [intrinsic function][6] merges the [Step Functions context object][7] (`$$`) with the original Lambda's input payload (`$`). Fields of the original payload overwrite the Step Functions context object if their keys are the same.
+
+   **Example**:
+
+{{< highlight json "hl_lines=4-7" >}}
```

Comment:
Did you want the examples lined up under Example? As it is, it's aligned to the main page. https://a.cl.ly/RBuJJrp1

Reply:
yep, known issue. the `highlight` blocks don't render properly when indented.